es6-modules/src/components/todo-list.js

```diff
@@ -0,0 +1,50 @@
+import createElement from '../lib/create-element';
```

Comment:
Here you want to import from the `lib/index.js`. We will go over tonight why one is preferred over the other.

```JAVASCRIPT
import { createElement } from '../lib';
```